test/test_operations.py

```diff
@@ -965,6 +965,9 @@ def func(a, b):
     b = torch.ones([2, 2])
     self.runAtenTest((a, b), func)
 
+  @unittest.skipIf(
```

Comment:
Instead of skipping the tests here, maybe you can utilize the env and have different assertEquals.

Reply:
Forgot the details. Thanks for pointing it out. Then, let's skip.